### ISSUE TITLE
Remove unused features, ui elements that clutter the visualizer

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/three.js#4]
The visualizer is based on the three.js/editor but strives to add features to make it more user friendly to OpenSim application users, and remove unused/unnecessary features that could confuse or overwhelm. This issue is a place holder for the latter.

**Comment by aymanhab** [comment]
"Fog" comes to mind as one unused feature, as well as the translate/rotate/scale of objects that are not supported.

**Comment by tkuchida** [comment]
There are three options under Settings->Renderer. The CanvasRenderer and SVGRenderer do not display the ground plane or sky cube and are very slow, so perhaps the option should be removed?

**Comment by aymanhab** [comment]
If we remove the remaining renderer options we should remove the dropdown altogether, which would be good. Don't know though if these can be handy in generating eps files or similar. Main use is if RaytraceRenderer can work to produce raytraced rendering of scene.

**Comment by tkuchida** [comment]
The "Compute Vertex Normals" button that appears when you click on a body doesn't seem to do anything.

**Comment by aymanhab** [comment]
CanvasRenderer claims to be useful when webgl is not supported, does it make sense to leave behind @tkuchida @chrisdembia ? I think not.

**Comment by aymanhab** [comment]
Right now, the Object pane has a "User Data" field, but I don't understand what it contains or what it could be used for. Remove? from issue #21, will consolidate here and close it.

**Comment by chrisdembia** [comment]
> CanvasRenderer claims to be useful when webgl is not supported, does it make sense to leave behind @tkuchida @chrisdembia ? I think not.

I agree. Let's leave it out for now and wait to see if users don't have webgl support.
